databases/week1/w1.sql

```diff
@@ -0,0 +1,17 @@
+SELECT COUNT(task.id) FROM task;
+SELECT * FROM task;
+SELECT COUNT(task.id) FROM task WHERE due_date IS NULL ;
+SELECT * FROM task JOIN status ON task.status_id = status.id WHERE status.name like "done";
+SELECT * FROM task JOIN status ON task.status_id = status.id WHERE status.name not like "done";
```

Comment:
The SQL statements are case sensitive so if the value of the status.name is "Done" then this query will not return the correct value. 

You can also use `= "Done"` and `!= "Done"`